.travis.yml

```diff
@@ -14,3 +14,9 @@ matrix:
     - rvm: jruby-9.1.6.0
 before_install:
   - gem --version
+
+script:
+   - bundle exec rubocop --version
+   - bundle exec rubocop -D .
```

Comment:
`.` does nothing and `-D` should go into .rubocop.yml
preferably make this parallel ... whipped up a PR for that https://github.com/fakefs/fakefs/pull/367